src/EnergyPlus/api/state.py

```diff
@@ -85,7 +85,7 @@ def new_state(self) -> c_void_p:
 
         :return: A pointer to a new state object in memory
         """
-        return self.api.stateNew()
+        return self.api.stateNewPython()
```

Comment:
state_new in python calls stateNewPython... That's unfortunate to add another extern C method, but given the lack of support in C for default argument values...

Reply:
Yeah, it's not a big deal to me at all.  This is hidden within the API anyway, so all good.